enable/tests/wx/mouse_wheel_test_case.py

```diff
@@ -63,13 +63,18 @@ def test_vertical_mouse_wheel(self):
         # validate results
         self.assertEqual(self.tool.event.mouse_wheel_axis, 'vertical')
         self.assertAlmostEqual(self.tool.event.mouse_wheel, 5.0/3.0)
-        self.assertEqual(self.tool.event.mouse_wheel_delta, (0, 200))
+
+        # "Expected failure" here
+        # The expected value is probably wrong.
+        # When the test was written, the expected value was (0, 200)
+        # (0, 200) would match Qt test (enthought/enable#458)
+        self.assertEqual(self.tool.event.mouse_wheel_delta, (0, 1.0))
 
     def test_horizontal_mouse_wheel(self):
         import wx
 
         # create and mock a mouse wheel event
-        wx_event = wx.MouseEvent(mouseType=wx.wxEVT_MOUSEWHEEL)
+        wx_event = wx.MouseEvent(wx.wxEVT_MOUSEWHEEL)
```

Comment:
The argument name was changed to `mouseEventType`. I could use named argument here too.